docs/index.md

```diff
@@ -16,115 +22,237 @@
+
+
+UMAP: UMAP reduced the pairwise distance matrix (Figure 1).
+The UMAP pairwise vs euclidean distance plot gave a pearson coefficient of .471.
+The data points did splay out from the LOESS line, but far less so compared to t-SNE - the more distinguishing characteristics were the clusters shown in the scatter plot itself, revealing UMAP’s approach to visualizing local and global patterns equally (Figure 2).
+The ratio between the medians given by the within vs between violin plot for UMAP was 9:1, revealing tightly clustered data.
+The data points from different clades were far apart and incredibly densely packed.
+The density of the within clade violin plot is at much lower euclidean distances than the density of the between clade violin plot, revealing that it is extremely easy to discern the between vs within clade measure given a euclidean distance (corroborating the observation of UMAP’s densely packed and compartmentalized embedding of these data points) (Figure 3).
+The embedding did translate the data differently than the rendering of the tree, giving much more euclidean space between points that were more genetically diverse and packing genetically similar strains very close together, if not stacked.
+This pattern is clearly shown in the pairwise vs euclidean graph for UMAP, as the LOESS line begins to plateau as the genetic distance gets larger and larger, revealing that UMAP sets a threshold for genetic diversity and keeps points above that threshold very far apart in euclidean space.
+This is different from other embeddings, because PCA, MDS, and t-SNE do not set a threshold for genetic diversity, and instead scale the data based on the largest genetic distance disparity (this makes UMAP’s embedding so densely clustered).
+
+
+## Figure One
+
+<iframe src="https://blab.github.io/cartography/FullLinkedChartClickableFlu.html" style="width: 1200px; height: 400px;" frameBorder="0"></iframe>
+
+## Figure Two
+
+![](FullScatterplotFlu.png)
+
+## Figure Three
+![](FullViolinPlotFlu.png)
+
+
+
+## Zika:
+
+Zika: Zika in this project is used as a test case. 
+As zika genes all exist on the same genome and zika’s genome is considerably longer than H3N2 Influenza genome, the genetic diversity of zika, while low enough to create phylogenetic transmission tree, clades are hard to distinguish from each other - zika transmission trees are often colored by region and country for this reason. 
+It can be reasonably assumed that zika is a good test case for Cartography.
+
+## SUMMARY OF RESULTS FOR ZIKA: 
+
+PCA: The data was normalized and scaled, and PCA was run on it (Figure 4). 
```

Comment:
After reading through these paragraphs about each method's results for flu and Zika, I'd like to suggest a different formula for presenting the results for each method:

1. What does the embedding look like qualitatively? Do there appear to be visually identifiable clusters in the data? Do all the points group together in a single mass? Reference the appropriate panels from Figures 1 and 4 here for flu and Zika, respectively (e.g., Figure 1B for flu PCA results).

2. What is the qualitative relationship between Euclidean and genetic distance (i.e., describe the general patterns you observe. Is the relationship linear across all genetic distances or is it initially linear and then it changes at higher distances)? Reference the appropriate panels from Figures 2 and 5. Support this statement with the R^2 value. What does this result confirm about the local vs. global behavior of the embedding method?

3. How well does the Euclidean distance between viral genomes distinguish viruses from the same or different clades? Reference the appropriate panels from Figures 3 and 6. Support these statements with the p-values from the permutation tests (see above) comparing within- and between-clade distances per embedding. Do any of the embedding methods distinguish viruses from different clades better than the genetic distance? We'll need to support these statements with additional permutation tests that we'll need to discuss offline.